Online/Entity/OnlinePhysicalObject.cs

```diff
@@ -615,5 +617,13 @@ public void Explode(Vector2 pos)
                     RainMeadow.Error($"unknown explode {this}"); return;
             }
         }
+
```

Comment:
imo if you're already making a state then might as well sync it as state
set bool true while hazer is hazing, unset once smoke stops